bolt-contracts/src/contracts/BoltChallenger.sol

```diff
@@ -98,92 +98,130 @@ contract BoltChallenger is IBoltChallenger {
 
     // ========= CHALLENGE CREATION =========
 
-    // Q: should we add a commit-reveal scheme to prevent frontrunning to steal slashing rewards?
+    /// @notice Open a challenge against a bundle of committed transactions.
+    /// @dev The challenge bond must be paid in order to open a challenge.
+    /// @param commitments The signed commitments to open a challenge for.
     function openChallenge(
-        SignedCommitment calldata commitment
+        SignedCommitment[] calldata commitments
     ) public payable {
+        if (commitments.length == 0) {
+            revert EmptyCommitments();
+        }
+
         // Check that the challenge bond is sufficient
-        if (msg.value < CHALLENGE_BOND) {
+        uint256 expectedBond = COMMITMENT_BOND * commitments.length;
```

Comment:
After thinking about it you're right, it's just anti-spam so it doesn't need to scale with commitments. 
I was just thinking that since resolving bigger challenges requires more gas you'd want a bigger refund to defend.

So far I haven't made a test of the incremental gas cost to include 2+ signed commitments in the resolution tho.